aodndata/moorings/products_handler.py

```diff
@@ -236,44 +258,64 @@ def _make_aggregated_timeseries(self):
                 raise InvalidFileContentError("No files to aggregate for {var}".format(var=var))
             self.logger.info("Aggregating {var} ({n} files)".format(var=var, n=len(input_list)))
 
-            product_url, errors = main_aggregator(input_list, var, self.product_site_code, input_dir=self.temp_dir,
-                                                  output_dir=self.products_dir,
-                                                  download_url_prefix=DOWNLOAD_URL_PREFIX,
-                                                  opendap_url_prefix=OPENDAP_URL_PREFIX
-                                                  )
+            product_url, errors = main_aggregator(input_list, var, self.product_site_code,
+                                                  **self.product_common_kwargs)
             self._log_excluded_files(errors)
+            self._add_to_collection(product_url)
+            self._cleanup_previous_version(os.path.basename(product_url))
+
+    def _make_velocity_aggregated_timeseries(self):
+        """Generate the velocity aggregated timeseries product and add to file_collection."""
+
+        # Filter input list to just the velocity files, all of which will have the variable 'VCUR'
+        input_list = [f for f, f_vars in self.input_file_variables.items()
+                      if 'VCUR' in f_vars
```

Comment:
I would change the comment to highlight why VCUR is used as a matching filter instead.

Reply:
Would this be enough to make it clear? (If not, then I'm not sure what you mean)
```python
# Filter input list to just the velocity files, all of which will have
# the variable 'VCUR' (= "northward_sea_water_velocity")
```

Reply:
It was more like a warning because of the simple matching filter - I imagine you are using VCUR as a placeholder for ``` if 'UCUR' in ... and 'VCUR' in ... and ... other possible conditions matched by files containing VCUR variables ```